src/Components/UpcomingTable/UpcomingTable.tsx

```diff
@@ -559,39 +607,31 @@ export const UpcomingTable: React.FunctionComponent<UpcomingTableProps> = (
   return (
     <React.Fragment>
       {toolbar}
-      <Table aria-label="Selectable table">
+      <Table
+        aria-label="Expandable table"
+        variant={isExampleCompact ? 'compact' : undefined}
+      >
         <Thead>
           <Tr>
-            <Th width={10}>{columnNames.name}</Th>
+            <Th screenReaderText="Row expansion" />
+            <Th width={30}>{columnNames.name}</Th>
+            <Th width={10}>{columnNames.type}</Th>
             <Th width={10}>{columnNames.release}</Th>
-            <Th width={10}>{columnNames.date}</Th>
+            <Th width={20}>{columnNames.date}</Th>
```

Comment:
Same kind of thing here. Let's loop through the `columnNames` to create our header row.